Core/include/Acts/TrackFitting/detail/GsfActor.hpp

```diff
@@ -235,9 +235,8 @@ struct GsfActor {
     }
 
     for (auto cmp : stepper.componentIterable(state.stepping)) {
-      auto singleState = cmp.singleState(state);
-      cmp.singleStepper(stepper).transportCovarianceToBound(
-          singleState.stepping, surface);
+      cmp.singleStepper(stepper).transportCovarianceToBound(cmp.state(),
```

Comment:
I think this previously created a copy, and now it passes a reference. I'm assuming this is safe?

Reply:
`transportCovarianceToBound` should only depend on a const reference